### PR TITLE
PCHR-343 - changed default front page for civihr_admin and added upgr…

### PR DESCRIPTION
**Pull request by Agares4**
…ader for login_destination

Default login_destination has to be changed in buildkit (~~I'll send a PR for that~~ https://github.com/civicrm/civicrm-buildkit/pull/209)

Relates to: https://github.com/compucorp/civihr-tasks-assignments/pull/6

### CODE REVIEW

**Comment by AkA84** [comment]
Foo bar baz

File: civihr_employee_portal/civihr_employee_portal.module [hunk 1]
```diff
@@ -5338,4 +5338,4 @@ function _getManagerContacts($contact_id) {
     }
 
     return $assigned_manager_contact_ids;
-}
+}
```
Comment (AkA84) [inline civihr_employee_portal/civihr_employee_portal.module]: Unnecessary change. Also make sure to have your IDE set so that on file save it makes sure there is a new line at the end of the file

**Comment by coldrunKacper** [comment]
Reviewed and approved.
